playbooks/lb_monitoring_setup.yaml

```diff
@@ -1,132 +1,24 @@
 ---
-- name: Prepare controller variables
+- name: Prepare main network with the controller host
   import_playbook: bastion_setup.yaml
 
-- name: Prepare variables
+- name: Create key pair for lb monitoring
   hosts: local
-  vars:
-    scenario_name: lb_monitoring
-    infra_state: present
-    terraform_variables:
-      network_cidr: "{{ terraform_network_cidr }}"
-      network_id: "{{ main_network_id }}"
-      router_id: "{{ main_router_id }}"
-      subnet_id: "{{ main_subnet_id }}"
-  roles:
-    - build_infrastructure
-  tasks:
-    - name: Register build result
-      set_fact:
-        lb_control_instance_ip: "{{ tf_output.outputs['lb_ctrl_ip'].value }}"
-        loadbalancer_public_ip: "{{ tf_output.outputs['lb_fip'].value }}"
-        lb_node_ips: "{{ tf_output.outputs['lb_ecs_local_ips'].value }}"
-    - name: Register nodes
-      add_host:
-        name: "{{ lb_control_instance_ip }}"
-        groups: control
-        ansible_ssh_common_args: >
-          -o UserKnownHostsFile=/dev/null
-          -o ProxyCommand='ssh -W %h:%p
-          -q linux@{{ controller_public_ip }}
-          -i {{ local_private_key }}
-          -o StrictHostKeyChecking=no
-          -o UserKnownHostsFile=/dev/null'
-          -o PasswordAuthentication=no
-
-    - name: Register nodes
-      add_host:
-        name: "{{ item }}"
-        groups: gatewayed
-        ansible_ssh_common_args: >
-          -o UserKnownHostsFile=/dev/null
-          -o ProxyCommand='ssh -W %h:%p
-          -q linux@{{ controller_public_ip }}
-          -i {{ local_private_key }}
-          -o StrictHostKeyChecking=no
-          -o UserKnownHostsFile=/dev/null'
-          -o PasswordAuthentication=no
-      loop: "{{ lb_node_ips }}"
-
-- name: Wait for hosts
-  hosts: gatewayed
-  gather_facts: no
   tasks:
-    - name: Wait for host to be up
-      wait_for_connection:
-        timeout: 120
-
-- name: Start Server
-  hosts: gatewayed
-  become: yes
-  roles:
-    - too_simple_server
-  vars:
-    debug: true
-    server_port: 80
+    - include_vars: ./vars/lb_monitoring.yaml
```

Comment:
instead of this you can use
- name: Create key pair for lb monitoring
  hosts: local
  vars_files:  
    - ./vars/lb_monitoring.yaml
  tasks:
...